files/galaxy/tpv/destinations.yml.j2

```diff
@@ -286,9 +286,7 @@ destinations:
       - if: tool.id.startswith("toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/alphafold") and helpers.tool_version_gte(tool, "2.1.2+galaxy0") and helpers.tool_version_lt(tool, "2.3")
         env:
           ALPHAFOLD_DB: "/storage/brno11-elixir/projects/alphafold/alphafold.db-2.2"
-        params:
-          singularity_volumes: "$job_directory:ro,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/cvmfs/data.galaxyproject.org:ro,$SCRATCHDIR,/storage/praha5-elixir/home/galaxyeu:/home/galaxyeu,$ALPHAFOLD_DB:/data/db/databases/alphafold_databases/2.2:ro"
```

Comment:
I assume you have things set up in a way that the params from the destination itself will work.

Reply:
Not exactly sure what you are trying to point out here

Reply:
Nothing really, I believe this comment was not needed.